services/dynamodb/src/main/java/software/amazon/awssdk/services/dynamodb/DynamoDbRetryPolicy.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.dynamodb;
+
+import java.time.Duration;
+import software.amazon.awssdk.annotations.SdkProtectedApi;
+import software.amazon.awssdk.awscore.retry.AwsRetryPolicy;
+import software.amazon.awssdk.core.retry.RetryPolicy;
+import software.amazon.awssdk.core.retry.SdkDefaultRetrySetting;
+import software.amazon.awssdk.core.retry.backoff.BackoffStrategy;
+import software.amazon.awssdk.core.retry.backoff.FullJitterBackoffStrategy;
+
+/**
+ * Default retry policy for DynamoDB Client.
+ */
+@SdkProtectedApi
+public final class DynamoDbRetryPolicy {
+
+    /** Default max retry count for DynamoDB client **/
+    private static final int DEFAULT_MAX_ERROR_RETRY = 10;
```

Comment:
There was a mention about this value being problematic in one of trevor's email. We should discuss if we need to lower it.

Reply:
Hmmm, good call, will take a look